### DISCUSSION TITLE
I accidentally cancelled the setup while not yet adding an admin

### DISCUSSION THREAD

**Discussion by Perladodaniel** [Pryaxis/TShock#2597] [Q&A]
So i accidentally closed it by doing another /setup and the answer to getting it again is getting a fresh start which will delete the data of my players characters is there any way to avoid this?

**Comment by GustavoSchip** [comment]
Make sure to make an account "register [passwd]"
Run this with the character in-game.

Then in the server console, use "/user group [user] owner"

**Reply by GustavoSchip** [reply]
U dont have to start over.
